CMakeLists.txt

```diff
@@ -60,8 +83,8 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR}
                     ${AASDK_INCLUDE_DIRS}
                     ${BCM_HOST_INCLUDE_DIRS}
                     ${ILCLIENT_INCLUDE_DIRS}
-                    ${include_directory})
-								
+                    ${include_directory}
+ 		    ${GST_INCLUDE_DIRS})
```

Comment:
this should be above `include_directory` and fix the indentation (and add a newline before `link_directories`)